<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>person</class>
 <widget class="QMainWindow" name="person">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1366</width>
    <height>768</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1366</width>
    <height>768</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1366</width>
    <height>768</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="wall">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1371</width>
      <height>771</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/pages/Resource/page3.jpg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="PersonalInfo_Widget" native="true">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>4118</x>
      <y>20</y>
      <width>1321</width>
      <height>591</height>
     </rect>
    </property>
    <widget class="QPushButton" name="Button_logout">
     <property name="geometry">
      <rect>
       <x>910</x>
       <y>300</y>
       <width>220</width>
       <height>220</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/button/Resource/buttom_logout.bmp);


</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="Button_Author">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>300</y>
       <width>220</width>
       <height>220</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/button/Resource/button_about.jpg);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_personalInfo">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>240</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;黑体&quot;;
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>个人信息</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Button_personalInfo">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>0</y>
       <width>220</width>
       <height>220</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/button/Resource/button_person.jpg);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="Button_setting">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>300</y>
       <width>220</width>
       <height>220</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/button/Resource/button_setting.jpg);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="Button_privacy">
     <property name="geometry">
      <rect>
       <x>910</x>
       <y>0</y>
       <width>220</width>
       <height>220</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/button/Resource/button_privacy.jpg);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="Button_money">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>0</y>
       <width>220</width>
       <height>220</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/button/Resource/button_money.jpg);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_privacy">
     <property name="geometry">
      <rect>
       <x>960</x>
       <y>240</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;黑体&quot;;
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>账户与隐私</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_logout">
     <property name="geometry">
      <rect>
       <x>970</x>
       <y>540</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;黑体&quot;;
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>注销账户</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_anthor">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>540</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;黑体&quot;;
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>关于作者</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_money">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>240</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;黑体&quot;;
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>钱包</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_setting">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>540</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;黑体&quot;;
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>系统设置</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="Buy">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>660</y>
      <width>291</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;黑体&quot;;
background-color: rgba(255, 255, 255, 50);
border-image: url(:/button/Resource/button_select.jpg);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>预定车票</string>
    </property>
   </widget>
   <widget class="QWidget" name="Buy_widget" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>1321</width>
      <height>591</height>
     </rect>
    </property>
    <widget class="QPushButton" name="pushButton_7">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>480</y>
       <width>531</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 24pt &quot;楷体&quot;;
border-image: url(:/button/Resource/button_search.jpg);
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>探索车票</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Button_start">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>90</y>
       <width>231</width>
       <height>111</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(255, 255, 255, 120);
font: 16pt &quot;楷体&quot;;</string>
     </property>
     <property name="text">
      <string>始发站

北京南站</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Button_end">
     <property name="geometry">
      <rect>
       <x>770</x>
       <y>90</y>
       <width>231</width>
       <height>111</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(255, 255, 255, 120);
font: 16pt &quot;楷体&quot;;</string>
     </property>
     <property name="text">
      <string>终到站

天津站</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Button_switch">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>60</y>
       <width>160</width>
       <height>160</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(255, 255, 255, 120);
font: 16pt &quot;楷体&quot;;</string>
     </property>
     <property name="text">
      <string>←
→</string>
     </property>
    </widget>
    <widget class="QWidget" name="station_list" native="true">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>200</y>
       <width>291</width>
       <height>401</height>
      </rect>
     </property>
     <widget class="QLineEdit" name="searchLine">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>261</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(255, 255, 255, 220);
font: 16pt &quot;楷体&quot;;</string>
      </property>
     </widget>
     <widget class="QListWidget" name="list_stations">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>261</width>
        <height>301</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(255, 255, 255, 220);
font: 16pt &quot;楷体&quot;;</string>
      </property>
     </widget>
     <zorder>list_stations</zorder>
     <zorder>searchLine</zorder>
    </widget>
    <widget class="QPushButton" name="pushButton_13">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>230</y>
       <width>661</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 16pt &quot;微软雅黑&quot;;
background-color: rgba(255, 255, 255, 120);</string>
     </property>
     <property name="text">
      <string>10月12日 星期天</string>
     </property>
    </widget>
    <widget class="QCalendarWidget" name="calendarWidget">
     <property name="geometry">
      <rect>
       <x>700</x>
       <y>320</y>
       <width>301</width>
       <height>236</height>
      </rect>
     </property>
    </widget>
    <zorder>pushButton_7</zorder>
    <zorder>Button_start</zorder>
    <zorder>Button_end</zorder>
    <zorder>Button_switch</zorder>
    <zorder>pushButton_13</zorder>
    <zorder>calendarWidget</zorder>
    <zorder>station_list</zorder>
   </widget>
   <widget class="QPushButton" name="Trip">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>660</y>
      <width>291</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;黑体&quot;;
background-color: rgba(255, 255, 255, 50);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>行程</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Order">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>660</y>
      <width>291</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;黑体&quot;;
background-color: rgba(255, 255, 255, 50);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>订单管理</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Person">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>660</y>
      <width>291</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt '黑体';
background-color: rgba(255, 255, 255, 50);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>我</string>
    </property>
   </widget>
   <widget class="QWidget" name="Trip_widget" native="true">
    <property name="geometry">
     <rect>
      <x>1386</x>
      <y>20</y>
      <width>1321</width>
      <height>591</height>
     </rect>
    </property>
    <widget class="QPushButton" name="pushButton_8">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>230</y>
       <width>761</width>
       <height>171</height>
      </rect>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="Order_widget" native="true">
    <property name="geometry">
     <rect>
      <x>2752</x>
      <y>20</y>
      <width>1321</width>
      <height>631</height>
     </rect>
    </property>
    <widget class="QListWidget" name="listWidget">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>50</y>
       <width>1231</width>
       <height>521</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(255, 255, 255, 80);
font: 25 14pt &quot;Microsoft YaHei UI Light&quot;;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_none">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>260</y>
       <width>151</width>
       <height>101</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 22pt &quot;楷体&quot;;</string>
     </property>
     <property name="text">
      <string>暂无订单</string>
     </property>
    </widget>
   </widget>
   <zorder>wall</zorder>
   <zorder>Buy</zorder>
   <zorder>Trip</zorder>
   <zorder>Order</zorder>
   <zorder>Person</zorder>
   <zorder>PersonalInfo_Widget</zorder>
   <zorder>Buy_widget</zorder>
   <zorder>Trip_widget</zorder>
   <zorder>Order_widget</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
